administering.md

```diff
@@ -0,0 +1,3 @@
+# Administering Application Service Adapter
+
+This section contains documentation related to administering an Application Service Adapter installation. It includes documentation covering system logs and metrics, backup and restore strategies, certificate rotation, and more.
```

Comment:
This page is a bit bare bones for now. Open to any suggestions for additional content or thoughts on whether or not this should link out to the docs within its section. Right now that felt a little redundant to the TOC and sidebar, though.